lib/devise_otp_authenticatable/controllers/helpers.rb

```diff
@@ -27,7 +27,7 @@ def otp_t()
       end
 
 
-      def trusted_devices_enabled?
+      def trusted_devices_enabled?(resource=resource)
```

Comment:
Yes because in `DeviseOtp::CredentialsController#update` `resource` is only available locally.  But now that you mention it maybe there is a better way to approach this.  I wonder if there is a standard way to set the `resource` such that it is accessible from other methods.